.github/ISSUE_TEMPLATE/20-feature-request.yml

```diff
@@ -0,0 +1,37 @@
+name: "Feature Request"
+description: Suggest a new feature for the package
+body:
```

Comment:
Miss the title/label sections here?

Reply:
The title for the bug had a prefix "[Bug] " to make the issue more evident as a bug. The others will have a default empty title. Same for the label.